docs/integrations/sumo-apps/enterprise-search-audit.md

```diff
@@ -32,28 +32,28 @@ To install the app:
 This page has information about the pre-built dashboards for Enterprise Search Audit App and a description of each of the app dashboards.
 
 
-### Search Overview
+### Enterprise Search Audit - Search Overview
 The Enterprise Search Audit - Search Overview dashboard provides a high-level overview of search usage within your account. The panels also display current search usage in terms of the total number of queries, types of query, and other query characteristics run by users.
 
 Use this dashboard to:
 * To quickly monitor searches run in your account, including by search type.
 * To get insights into query statistics, time ranges, and query failure rate.
 
-<img src={useBaseUrl('img/integrations/sumo-apps/ESA-SearchOverview.png')} alt="test" />
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="test" />
```

Comment:
I'd change the alt text to match the images for all four of these dashboards images

```suggestion
<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Enterprise-Search-Audit/Enterprise-Search-Audit-Search-Overview.png')} alt="Enterprise Search Audit - Search Overview" />
```